book/conclusion.md

```diff
@@ -1,4 +1,4 @@
 # Conclusion 
 
-In this work we used a simple clustering technique such as k-Means to identify highly reproducible functional parcellation at the full brain level, even when applied on short fMRI time series (a few minutes). We extended the DYPAC method to identify the main states of these dynamic parcellations, and demonstrated that their spatial distributions had high compression quality between the reduced and the raw signal. We also found different groups of dynamic states grouped by their reproducibility. The formalization of the parcellation problem as a dynamic problem would have further implications on the neuroimaging analyses, including many graph-based neuroimaging analyses. This opens new research directions to explore after replacing the static parcellations by the dynamic parcellations. Also, these parcellations would help studying the rich dynamic interactions between functional brain networks. 
+In this work we used a simple data-driven analysis such as k-Means to identify highly reproducible functional parcellation at the full brain level, even when applied on short fMRI time series (a few minutes). We extended the DYPAC method to identify the main states of these dynamic parcellations, and demonstrated that their spatial distributions had high compression quality between the reduced and the raw signal. We also found different groups of dynamic states grouped by their reproducibility. The formalization of the parcellation problem as a dynamic problem would have further implications on the neuroimaging analyses, including many graph-based neuroimaging analyses. This opens new research directions to explore after replacing the static parcellations by the dynamic parcellations. Also, these parcellations would help studying the rich dynamic interactions between functional brain networks. 
```

Comment:
```suggestion
In this TOTO we used a simple data-driven analysis such as k-Means to identify highly reproducible functional parcellation at the full brain level, even when applied on short fMRI time series (a few minutes). We extended the DYPAC method to identify the main states of these dynamic parcellations, and demonstrated that their spatial distributions had high compression quality between the reduced and the raw signal. We also found different groups of dynamic states grouped by their reproducibility. The formalization of the parcellation problem as a dynamic problem would have further implications on the neuroimaging analyses, including many graph-based neuroimaging analyses. This opens new research directions to explore after replacing the static parcellations by the dynamic parcellations. Also, these parcellations would help studying the rich dynamic interactions between functional brain networks. 
```

Reply:
thanks for your input. I'll think about it.